src/RadixDlt.NetworkGateway.GatewayApi/examples.md

```diff
@@ -0,0 +1,446 @@
+# How to use Gateway
```

Comment:
I left it as seprate file but we have to move it to gateway-api-schema.yaml under examples section.